sample-app/src/main/java/com/sinch/sample/verification/verifications/Start.java

```diff
@@ -34,10 +35,14 @@ public void run() {
     StartVerificationRequestParameters.Builder<?> builder;
 
     if (method != VerificationMethodType.CALLOUT) {
-      builder = StartVerificationRequestParameters.builder(method);
+      builder = StartVerificationCalloutRequestParameters.builder();
```

Comment:
I'm not sure to understand the logic here: it will always be a `callout` verification either on the line 38, either on line 40. The lines 42 and 44 seem unreachable

Reply:
Yes.
Sample purpose: just change the `method` assignation from line 33 to change the request to be sent to server
Body of function is for educational and highlight how to handle dedicated initialization.

Reply:
Right, I do not remember where I read/listen this a long time ago: `the "cut and paste" function was the worst invention`
I have to agree 😞

Reply:
🙌